tools/teal/examples/dynamicfee.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 # produce TEAL assembly for a delegated logic signature on a dynamic-fee transaction and compile it (note the required lease value)
-python driver.py dynamic-fee --cls AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAY5HFKQ --rcv NBH2CNQCPV7S2ZOSWVQ7JLFO7W5JERULPSGYUP4ZAEOLSVMTYODHVY37VQ --fv 170500 --lv 171500 --amt 100000 --x uFVDhjBpkpKQ8sZaau0qsDsf0eW3oXFEn1Ar5o39vkk= > dynamic.teal
+algotmpl -d `git rev-parse --show-toplevel`/tools/teal/templates dynamic-fee --cls AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAY5HFKQ --to NBH2CNQCPV7S2ZOSWVQ7JLFO7W5JERULPSGYUP4ZAEOLSVMTYODHVY37VQ --fv 170500 --lv 171500 --amt 100000 --lease uFVDhjBpkpKQ8sZaau0qsDsf0eW3oXFEn1Ar5o39vkk= > dynamic.teal
 goal clerk compile -a EVADN3MAXLTUAZFLJNIN7RD7WCNEPTB37GJMD5OACMVBKTGFXYYFMS5IT4 -s -o dynamic.lsig dynamic.teal -d .
```

Comment:
It would be nice to replace these concrete addresses with variables so that some users could actually use these bash scripts (this perhaps improves readability as well)

Reply:
Yeah, probably the right thing to do is to replace these with end-to-end tests. The things in the example directory are purely "documentation" (which also is not exactly right). Maybe we can change the extension on the example directory so people don't execute the scripts and think that they work correctly.